src/app/routes/DropDown.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable import/no-named-as-default */
+/* eslint-disable no-unused-expressions */
+import React from 'react';
+import Logout from '../../feature/auth/logout/Logout';
+
+const DropDown = (props) => {
+  const { user: { userName } } = props;
+  return (
+    <div className="dropdown-menu">
+      <ul>
+        <li className="user-name">
+          {`@${userName}`}
+        </li>
+        <a href="/profile">
+          <li>
+          Profile
+          </li>
+        </a>
+        <a href="/Create">
+          <li>
+            Create article
+          </li>
+        </a>
+        <a href="/Create">
+          <li>
+            Bookmarked
+          </li>
+        </a>
+        <Logout />
+      </ul>
+    </div>
+  );
+};
+
+export default DropDown;
```

Comment:
Because these files are only related to nav that's why I could not put them in common folder. Our nav file has been in this folder since we started this project.

Reply:
That's true but these files are only related to nav that's why I could not put them in common folder. Our nav file has been in this folder since we started this project. So to avoid unnecessary conflict from our codebase I decided to not remove nav in that folder.